### ISSUE TITLE
[News] NetSparkle has moved to NetSparkleUpdater!

### ISSUE THREAD

**Issue by Deadpikle** [dei79/netsparkle#2]
EDIT: Update -- The latest NetSparkle code is available at https://github.com/NetSparkleUpdater/NetSparkle. Please join us!

Hey dei79,

I see that you've recently brought the NetSparkle code from CodePlex here to Github! 😀 I'm not sure if you're aware that the CodePlex project was forked a long time ago to [this repo by jimgraham](https://github.com/jimgraham/NetSparkle). He didn't keep updating it, so someone forked that repo, which was then forked again, etc. until, most recently, I have been maintaining and updating the code at my fork, which is located [here](https://github.com/Deadpikle/NetSparkle). I've been using NetSparkle for a work project and have been updating it with new features and such. I am not responsible for all of the new features that are available, as several are from one or another fork of the code. Myself and another of my coworkers have been working on more updates this week, so the develop branch is more up-to-date. (We're still working on updating documentation. That's WIP as I type.)

I'd hate to see multiple "latest" NetSparkle forks floating around. It'd be great to have one central NetSparkle repo, and whether that's my repo or yours or a new repo under a NetSparkle GitHub organization is inconsequential to me. Would you like to coordinate with me on having a new central repo? We could use my repo or have a new organization or similar (giant merge into this repo?). If we use my repo, I can email GitHub support to remove the "forked from" portion of the page, and then we can redirect from here to there or similar. I'm more than willing to give you access to my repo if that's what you'd like to do. I would have emailed you, but I didn't see contact info on your GitHub profile.

Please let me know! Make sure to highlight me @Deadpikle so that I am emailed when you reply.

Thanks!

-Deadpikle

**Comment by flemingm** [comment]
I also found: https://github.com/ashokgelal/NetSparkle

 @Deadpikle

**Comment by Deadpikle** [comment]
@flemingm

Yeah, that's another fork. Unfortunately, it hasn't been updated in over 4 years, so it's not really worth pointing people to. Not to toot my own horn, but I'm willing to accept and manage PRs and other issues over at [my repo](https://github.com/Deadpikle/NetSparkle), and I'll be updating the [NetSparkle NuGet](https://www.nuget.org/packages/NetSparkle/) eventually with a more up to date version (I have a version up at [NetSparkle.New](https://www.nuget.org/packages/NetSparkle.Net/) that I was using before being given access to the NetSparkle NuGet.)

**Comment by dei79** [comment]
Hello @Deadpikle,

yes I was the original author of this project but looking for a new coordinator. So should you be interested I'm happy to transfer this Repo to you. 

Regards
Dirk

**Comment by flemingm** [comment]
Multiple old repo -- with mostly dead is an issue.  I point out the other repository -- I should have said it has not be updated for a while.  

It's been forked several times -- checking members: https://github.com/ashokgelal/NetSparkle/network/members

So if we can consulate into one active repository it will improve support and fixes for all.  

@Deadpikle and @dei79.  

Deadpikle repo.  seem to Most active right now. and it was forked from the repository I quoted above. 

Maybe notify people that forked Dirk's repo that they should consider moving  to @Deadpikle and update the readme to tell people  which is the active repository.

Let me know how I can help.  I manage a group of programmers on Windows that are using an outdated version.  I was check what was current when I stumbled on these repository.

Mark

**Comment by Deadpikle** [comment]
@flemingm 👍 Makes sense! 😁 Let me know if your team needs help with the updates in the newer version. 

@dei79 @flemingm  Great!! I went ahead and emailed GitHub Support to see about the best way to do this. I'm not sure if moving everyone to my repo is best (and archiving this one?) or making a new organization and pointing everyone there is best or if some other idea is best. I don't want to lose any watchers/stars on either repo as presumably all those users would want to know of updates...but at the same time I don't want to lose the fork history either as that is sort of the project history too. I'm not sure what's best. Do you have both have any input on the best way to do this?

**Comment by flemingm** [comment]
Any response form GitHub support?

**Comment by Deadpikle** [comment]
@flemingm Whoops, sorry for not replying! I've been super busy, and NetSparkle work (besides any support requests) is on the back burner until August or so.

Yes, they did respond. The gist is that they could make my repo the base repo instead of a fork of a fork of a fork. However, I don't like this idea, because it makes history look weird.

So here's my current plan for NetSparkle:

1. Create a NetSparkle organization
2. Move my repo to the NetSparkle organization
3. Work on a NetSparkle 2.0
4. Release 2.0 to the more-official-looking NuGet and deprecate my current one (NetSparkle.New) 
5. Update this repo to point people to the new organization

That should remove people from having to search through a fork history to find the latest stuff and create an organization for when others want to help out with the project.

Thoughts/questions?

**Comment by Deadpikle** [comment]
Update after a long time:

The repo is now available at https://github.com/NetSparkleUpdater/NetSparkle. I'm still working on a 2.0 version, but it's coming along quite well so far. Feel free to hop over there and join!

Edit 20 April, 2021: 2.0 is out!
